### PR TITLE
Adds docker container

### PR DESCRIPTION
**Pull request by hovancik**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+RUN mkdir /bot
+WORKDIR /bot
+
+ADD Pipfile Pipfile.lock /bot/
+
+RUN pip install --upgrade pip
+RUN pip install pipenv
+RUN pipenv install --deploy --system
+
+ADD . /bot
```
Comment (neilhwatson) [inline Dockerfile:12]: You add file to the image here, but mount over later.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+version: '3'
+
+services:
+  bot:
+    build: .
+    volumes:
+      - .:/bot
```
Comment (neilhwatson) [inline docker-compose.yml:7]: No you mount over /bot, so the file ADDed by the docker file are hidden.

Comment (hovancik) [reply docker-compose.yml:7]: Interestingly, if I run `docker-compose run bot touch aloha.txt` I can see that file on my local machine.

Comment (hovancik) [reply docker-compose.yml:7]: So even more interestingly, this seems to be an Atom editor issue. From command line I can see the file, but Atom is not able to see it. Thanks for your time!

**Pull request merged**